trussworks/react-uswds
# feat: Add new bounded pagination component

# Summary

This PR adds the [USWDS Pagination component](https://designsystem.digital.gov/components/pagination/) in an MVP state (bounded). 

## Related Issues or PRs

#1543

Will need to be revisited to add unbounded pagination.

## How To Test

Check out this branch and run `yarn test` to execute tests and `yarn storybook` to see the new component in action.

### Screenshots (optional)
![Screen Shot 2022-03-14 at 10 12 14 AM](https://user-images.githubusercontent.com/19984283/158190072-bbe9504b-e3d5-4a60-a1ce-448707efd9fa.png)
![Screen Shot 2022-03-14 at 10 12 24 AM](https://user-images.githubusercontent.com/19984283/158190081-c0ab6bc3-6a58-43a8-969d-8b3db285c656.png)
